src/eventshub/dist/index.css

```diff
@@ -0,0 +1 @@
+@tailwind base;@tailwind components;@tailwind utilities;
```

Comment:
Shouldn't this be actual css...? Is postcss not running yet?

Reply:
Build vs publish thing perhaps? I have not RTFM yet on this however and could be mistaken.

Reply:
Postcss wasn't working due to how I defined the config. All tailwind/decanter things should be working as expected now!

Reply:
Huzzah!